prow/plugins/trigger/pull-request.go

```diff
@@ -76,6 +76,9 @@ func handlePR(c Client, trigger plugins.Trigger, pr github.PullRequestEvent) err
 			return fmt.Errorf("could not check membership: %s", err)
 		}
 		if member {
```

Comment:
Wouldn't it make more sense to to write a wrapper around `buildAll` and use it for all pull request events? The current approach means that we omit the building on drafts only for certain events where we implemented it, but not for others (reopening a PR, pushing to a PR, Labeling a PR, PR base change)

Reply:
The only one currently not implemented is Reopen.
I left that out on purpose with a reason that doesn't make sense, now that you mention it.
There already is buildAllIfTrusted which sounds like it, but i'm not sure why it only gets used in some places right now.
I'll look at it again.

Reply:
How are things going?

Reply:
I didn't have much time since last week, I'll have some more this one.
Can't wait to get this done.

Reply:
Okay, i cleaned it up a little.
There still is an additional draft check for create, to comment.
I decided to only send the skip comment on create to not spam it. If you want to, i can also send it on comment.

Reply:
Logging might be confusing now, as it logs that all jobs get started and then if the PR is a draft, that they get skipped.
I wouldn't see it as problem, but i can move the log message to buildAllButDrafts which would loose some details due to the different logs messages before buildAll.